apis/clusters/v1beta1/postgresql_types.go

```diff
@@ -622,3 +601,22 @@ func (pgs *PgStatus) DCsFromInstAPI(iDCs []*models.PGDataCentre) (dcs []*DataCen
 	}
 	return
 }
+
+func GetDefaultPgUserSecret(
```

Comment:
Because it is a separate function that does not use postgresql cluster fields and gets the name and namespace for the secret. So I think we can leave this as just a function and not a method.